src/lang/english.txt

```diff
@@ -93,37 +93,37 @@ STR_CARGO_SINGULAR_FIZZY_DRINK                                  :Fizzy Drink
+STR_QUANTITY_PASSENGERS                                         :{COMMA}{NBSP}Passenger{P "" s}
+STR_QUANTITY_COAL                                               :{WEIGHT_LONG} of Coal
+STR_QUANTITY_MAIL                                               :{COMMA}{NBSP}bag{P "" s} of Mail
+STR_QUANTITY_OIL                                                :{VOLUME_LONG} of Oil
+STR_QUANTITY_LIVESTOCK                                          :{COMMA}{NBSP}item{P "" s} of Livestock
+STR_QUANTITY_GOODS                                              :{COMMA}{NBSP}crate{P "" s} of Goods
+STR_QUANTITY_GRAIN                                              :{WEIGHT_LONG} of Grain
+STR_QUANTITY_WOOD                                               :{WEIGHT_LONG} of Wood
+STR_QUANTITY_IRON_ORE                                           :{WEIGHT_LONG} of Iron Ore
+STR_QUANTITY_STEEL                                              :{WEIGHT_LONG} of Steel
+STR_QUANTITY_VALUABLES                                          :{COMMA}{NBSP}bag{P "" s} of Valuables
+STR_QUANTITY_COPPER_ORE                                         :{WEIGHT_LONG} of Copper Ore
+STR_QUANTITY_MAIZE                                              :{WEIGHT_LONG} of Maize
+STR_QUANTITY_FRUIT                                              :{WEIGHT_LONG} of Fruit
+STR_QUANTITY_DIAMONDS                                           :{COMMA}{NBSP}bag{P "" s} of Diamonds
+STR_QUANTITY_FOOD                                               :{WEIGHT_LONG} of Food
+STR_QUANTITY_PAPER                                              :{WEIGHT_LONG} of Paper
+STR_QUANTITY_GOLD                                               :{COMMA}{NBSP}bag{P "" s} of Gold
+STR_QUANTITY_WATER                                              :{VOLUME_LONG} of Water
+STR_QUANTITY_WHEAT                                              :{WEIGHT_LONG} of Wheat
+STR_QUANTITY_RUBBER                                             :{VOLUME_LONG} of Rubber
+STR_QUANTITY_SUGAR                                              :{WEIGHT_LONG} of Sugar
+STR_QUANTITY_TOYS                                               :{COMMA}{NBSP}Toy{P "" s}
+STR_QUANTITY_SWEETS                                             :{COMMA}{NBSP}bag{P "" s} of Sweets
+STR_QUANTITY_COLA                                               :{VOLUME_LONG} of Cola
+STR_QUANTITY_CANDYFLOSS                                         :{WEIGHT_LONG} of Candyfloss
+STR_QUANTITY_BUBBLES                                            :{COMMA} Bubble{P "" s}
+STR_QUANTITY_TOFFEE                                             :{WEIGHT_LONG} of Toffee
+STR_QUANTITY_BATTERIES                                          :{COMMA} Batter{P y ies}
+STR_QUANTITY_PLASTIC                                            :{VOLUME_LONG} of Plastic
+STR_QUANTITY_FIZZY_DRINKS                                       :{COMMA} Fizzy Drink{P "" s}
 STR_QUANTITY_N_A                                                :N/A
```

Comment:
I really don't like this change. It's not helpful for understanding, and completely wrong as English grammar.

Reply:
Oh, and it breaks consistency with all NewGRF industry sets in a very conspicuous way.

Reply:
This one is commonplace in video game formatting: resource names are capitalized for emphasis. It helps visibility on certain menus. Moreover it avoids inconsistency with the fact these are capitalized in wagon names.

![image](https://user-images.githubusercontent.com/3621746/81869130-cb21f780-9573-11ea-92f8-f06f03c0d96b.png)